11ty/definitions/fr_FR/non-binaire.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Non-binaire
+slug: non-binaire
+speech: adj
+defined: true
+excerpt: de, relatif à, ou caractérisé par le fait d'être de n'importe quel genre qui ne correspond pas au binaire homme-femme ; un terme générique.
+---
+
+de, relatif à, ou caractérisé par le fait d'être de n'importe quel genre qui ne correspond pas au binaire homme-femme ; un terme générique.
+
+Par exemple, une personne non-binaire peut avoir un genre entre homme et femme, avoir plusieurs genres, aucun genre, un genre culturellement spécifique (ex : bispirituel).
+
+## Note
+
+Les personnes non-binaires peuvent aussi s'identifier ou non comme [transgenres](/definitions/fr_FR/transgenre).
+
+Bien que non-binaire soit couramment utilisé comme terme générique, le simple fait qu'une personne s'identifie à un terme compris dans ce terme générique n'implique pas nécessairement qu'elle s'identifie aussi comme non-binaire. Par exemple, une personne peut être agenre, mais ne pas s'identifier au terme "non-binaire".
```

Comment:
I would recommend shortening this sentence for clarity. Maybe:
```
Bien que "non-binaire" soit couramment utilisé comme terme générique, le simple fait qu'une personne s'identifie avec un terme générique n'implique pas nécessairement qu'elle s'identifie aussi comme non-binaire. Par exemple, une personne peut être agenre, sans s'identifier en tant que "non-binaire".
```

Reply:
That's great! I had a hard time trying to translate this one because it was quite complex (as you noted) but I think I was also trying to stick to the original text a little bit too much. Thanks for noticing my struggle! 🌟